Specs/Indy/1.15.0/Indy.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+
+  s.name         = "Indy"
+  s.version      = "1.15.0"
+  s.summary      = "Indy SDK iOS Wrapper - SSI Technologies"
+  s.homepage     = "https://www.hyperledger.org/use/hyperledger-indy"
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = { "Niall Shaw" => "niall.shaw@absa.africa" }
+
+  s.platform     = :ios, "10.0"
+
+  s.source       = { :http => "https://github.com/hyperledger/indy-sdk-react-native/releases/download/0.2.2/ios-indy-sdk-1.15.zip" }
+  s.framework    = "Indy"
+  s.source_files = "**/*.h"
+  s.vendored_frameworks = "IndySdk/Indy.framework"
```

Comment:
Does this need to change now that we renamed the zip file?

Reply:
Shouldn't it be 1.15.0 to standardise?

Reply:
I ended up creating a 1.15.0 **and** a 1.16.0 podspec (as it seemed Absa may have needed 1.15.0 specifically). 

> Shouldn't it be 1.15.0 to standardise?

Android was actually updated to 1.16.0 [recentlyish](https://github.com/hyperledger/indy-sdk-react-native/commit/169d6e5a3e1108d223bdfb897b9580e7cfd62528), so I think we'd like to standardize on 1.16.0 (especially since it's the most recent version of Indy available).